source/v0/TCPAsynch.cpp

```diff
@@ -53,6 +52,6 @@ TCPAsynch::~TCPAsynch()
 {
     _TCPSockets--;
     if (!_TCPSockets) {
-        _ticker.detach();
+        minar::Scheduler::cancelCallback(_tick_handle);
```

Comment:
might be a good idea to set _tick_handle to 0 / otherwise clear it after cancelling, just in case a programming error is introduced in the future which causes an attempt to free the handle twice.